.github/workflows/ci.yml

```diff
@@ -191,3 +191,14 @@ jobs:
         if: startsWith(runner.os, 'Windows') == false &&  matrix.skeleton == true
         working-directory: ${{ matrix.build-dir }}
         run: ./run_tests.sh --skeleton
+
+      - name: Install
+        run: |
+          cmake --build ${{ matrix.build-dir }} --target install
+          find install
```

Comment:
What does this `find install` do? Can't find it in GitHub Actions docs. :)

Reply:
There is no `tree`, so `find install` just prints `install` directory recursively

Reply:
Ah! I forgot this line is part of  shell script, and assumed it's some special GitHub Actions thing. My bad.